frontend/src/classes/TownController.test.ts

```diff
@@ -108,7 +108,7 @@ describe('TownController', () => {
     });
     it('Initializes the properties of the controller', () => {
       expect(testController.providerVideoToken).toEqual(townJoinResponse.providerVideoToken);
-      expect(testController.pets).toEqual(townJoinResponse.currentPets); // TODO:
+      expect(testController.pets).toEqual(townJoinResponse.currentPets);
```

Comment:
might want to add another line for the emotes - must have missed it in the emote branch